docs/API.md

```diff
@@ -895,19 +903,16 @@ The value of `"test_params"` is an object with the following properties:
 * `"config"`: A string, optional. The name of a *config profile*.
 * `"user_ip"`: A ..., optional.
 * `"user_location_info"`: A ..., optional.
-* `"priority"`: An integer, optional
-* `"queue"`: An integer, optional
+* `"priority"`: A *priorty*, optional
+* `"queue"`: An *queue*, optional
 
-*priotity*
+priority parameter
 
-This parameter is a free integer that will be used by the zonemaster testing backend daemon to sort the test requests from highest to lowest priority.
-This parameter will typically be used in a setup where a GUI will send requests to the bachend and would like to get response as soon as possible while at the same time using the idle time for background bach testing.
-The drawback of this setup will be that the GUI will have to wait for at least one background processing slot to become free (would be a few secods in a typical installation with up to 30 parallel zonemaster processes allowed)
+see the *priority* section
 
-*queue*
+queue parameter
```

Comment:
This is section is redundant since the data type is already pointed out in the params list above.